ynr/apps/moderation_queue/templates/moderation_queue/_photo-upload-image-form.html

```diff
@@ -21,7 +21,7 @@ <h2>Upload a photo from your computer</h2>
 
   <p>
     {% if queued_images %}
-    If you still want to upload another photo of {{ person.name }}, select an image from your computer (maximum size: 50 MB):
+    If you still want to upload another photo of {{ person.name }}, cancel the existing queued photos first.
```

Comment:
Is there a plan for cancelling uploaded photos? Feels like this perhaps should only be possible by the person that uploaded or a reviewer.

Reply:
Or are we going to allow a user to see the uploaded photo and replace it?

Reply:
Sym said, yes but not yet. I think this is just first step.

Reply:
More detail: Looking back at my notes, the 'Rejected' option in the moderation form is the accepted solution for now.

Reply:
Ok makes sense, however not sure we should say "cancel the existing queued photos first" if that is not possible. Something like this might be better? "Until the existing uploaded photo has been approved or rejected it is not possible to upload a new photo"